doc/ref_material/nest2_vs_3.rst

```diff
@@ -78,11 +78,9 @@ Functions related to connection
 |                                             | *is given by synapse_model,*                  |
 |                                             | *not model.*                                  |
 +---------------------------------------------+-----------------------------------------------+
-| nest.DataConnect(pre, params=None,          | *Use* nest.Connect(list, list,                |
+| nest.DataConnect(pre, params=None,          | *Use* nest.Connect(array, array,              |
```

Comment:
To avoid confusion with the built-in `array` module.
```suggestion
| nest.DataConnect(pre, params=None,          | *Use* nest.Connect(np.array, np.array,        |
```